packages/ibm-products-styles/src/components/Datagrid/styles/_datagrid.scss

```diff
@@ -156,6 +156,7 @@
   background-color: $background;
 }
 
+/* This section to be removed after support for slug dropped  - start */
```

Comment:
can we remove this section in this pr itself? and add corresponding aiLabel class names conditionally on presence of aiLabel || slug prop? can refer alex pr.

Reply:
Some adapters are using slug classes for customising the style for the values passes slug. Hence completely removing the classes could break such code. To be on the safer side it would be better to keep the slug classes until be completely remove the support for slug.

For now, I've left this as it is. I tried to refactor this section by adding aiLabel classes along with the slug classes. But this feels to0 bulky and not so readable. Since we are going to remove the slug classes in future anyway, it felt this would make more sense.